Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -86,6 +86,11 @@ @implementation FIRPhoneAuthProvider {
       @brief The auth instance used to for verifying the phone number.
    */
   FIRAuth *_auth;
+
+  /** @var _eventID
+      @brief The event ID associated with the a pending app verification request.
+   */
+  NSString *_eventID;
```

Comment:
This class should be state-less. Please capture the event ID string in the flow, rather than saving it in the iVar.

Reply:
Done.